docs/src/cfe_sb.dox

```diff
@@ -292,20 +292,21 @@
 
   The sequence counter for command messages is not altered by the software bus.
 
-  For telemetry messages sent with the #CFE_SB_SendMsg API, the software bus populates the packet sequence
-  header field for all messages. The first time a telemetry message is sent with a new Message ID,
-  the sequence counter field in the header is set to a value of one. For subsequent
-  sends of a message, the sequence counter is incremented by one regardless of the number of
-  destinations for the packet. After a rollover condition the sequence counter will be a
-  value of zero for one instance. The sequence counter is incremented in the #CFE_SB_SendMsg
-  API after all the checks have passed prior to the actual sending of the message. This
-  includes the parameter checks and the memory allocation check.
-  Note: The count is incremented regardless of whether there are any subscribers.
-
-  For telemetry messages sent with the #CFE_SB_PassMsg API the sequence counter is not incremented.
-  This method of message delivery is recommended for situations
-  where the sender did not generate the packet, such as a network interface application
-  passing a packet from a remote system to the local software bus.
+  For a telemetry message, the behavior is controlled via input parameters or API selection
```

Comment:
What "API selection" is used for this?

Reply:
It used to be via CFE_SB_SendMsg and CFE_SB_PassMsg.  CFE_SB_TransmitMsg just has an input parameter to control it.

Reply:
Updated, see 76a0951